<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>WdgRouteFilterPopup</class>
 <widget class="QWidget" name="WdgRouteFilterPopup">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>413</width>
    <height>120</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QFormLayout" name="formLayout">
   <item row="0" column="0">
    <widget class="QLabel" name="lDirection">
     <property name="text">
      <string>Direction:</string>
     </property>
    </widget>
   </item>
   <item row="0" column="1">
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QComboBox" name="cbDirection"/>
     </item>
     <item>
      <widget class="QToolButton" name="tbDirectionClear">
       <property name="text">
        <string>Clear</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item row="1" column="0">
    <widget class="QLabel" name="lFirstBusstop">
     <property name="text">
      <string>First Busstop:</string>
     </property>
    </widget>
   </item>
   <item row="1" column="1">
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <item>
      <widget class="QComboBox" name="cbFirstBusstop"/>
     </item>
     <item>
      <widget class="QToolButton" name="tbFirstBusstopClear">
       <property name="text">
        <string>Clear</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item row="2" column="0">
    <widget class="QLabel" name="lLastBusstop">
     <property name="text">
      <string>Last Busstop:</string>
     </property>
    </widget>
   </item>
   <item row="2" column="1">
    <layout class="QHBoxLayout" name="horizontalLayout_3">
     <item>
      <widget class="QComboBox" name="cbLastBusstop"/>
     </item>
     <item>
      <widget class="QToolButton" name="tbLastBusstopClear">
       <property name="text">
        <string>Clear</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
